frontend/src/pages/_events/slug/register/RegistrationSectionCustom/index.js

```diff
@@ -123,18 +123,15 @@ export default ({ section, onNext, nextLabel, onPrev, prevLabel, data }) => {
                         flexDirection="column"
                         alignItems="center"
                     >
-                        <Box maxWidth="600px">
+                        <Box>
                             <Markdown
                                 source={section.description}
                                 light
                                 alignCenter
                             />
                         </Box>
                         {section.conditional && (
-                            <Box
-                                maxWidth="600px"
-                                className={classes.radioGroupWrapper}
-                            >
+                            <Box className={classes.radioGroupWrapper}>
```

Comment:
Could it be fine to remove the className if it is not being used?

Reply:
Oh its actually related to some styling stuff so i didn't touch it:
``` js
radioGroupWrapper: {
        background: 'white',
        padding: theme.spacing(2),
        width: '100%',
        flexDirection: 'column',
        alignItems: 'center',
        justifyContent: 'center',
    },
```